environs/config/config.go

```diff
@@ -274,6 +274,14 @@ const (
 	// different state.
 	ModeKey = "mode"
 
+	// SSHAllowListKey is a comma separated list of CIDRs to allow ingress from
+	// to the ssh service on machines in this model
+	SSHAllowListKey = "ssh-allowlist"
```

Comment:
I think these are too long; cumbersome to remember at the CLI. I think they should be:
- `ssh-allow`
- `saas-ingress-allow`

Reply:
Happy to make these